.github/workflows/main.yml

```diff
@@ -127,12 +131,55 @@ jobs:
           path: /home/runner/work/netbeans/netbeans/nbbuild/build/test/commit-validation/results/
           retention-days: 1
 
+
+  sig-test:
+    name: Verify sigs/libs/licenses/RAT on Linux/JDK ${{ matrix.java }}
+    needs: base-build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '11' ]
+    steps:
+
+      - name: Set up JDK ${{ matrix.java }} 
+        uses: actions/setup-java@v2
+        with:
+          java-version: ${{ matrix.java }} 
+          distribution: 'zulu'
+
+      - name: Download Build
+        uses: actions/download-artifact@v3
+        with:
+          name: build
+
+      - name: Extract
+        run: tar --zstd -xf build.tar.zst
+
+      - name: Verify libs and licenses
+        run: ant $OPTS -quiet verify-libs-and-licenses -Dverify-libs-and-licenses.haltonfailure=true
+
+      - name: Check line endings and verify RAT report
+        if: success() || failure()
+        run: |
+          nbbuild/travis/check-line-endings.sh
+          ant -quiet build-source-config
+          mkdir scratch && cd scratch
+          unzip -qq ../nbbuild/build/release-src*
+          ant $OPTS -quiet rat -Drat-report.haltonfailure=true
+
+      - name: Run check-sigtests-release
+        if: success() || failure()
+        run: ant $OPTS -quiet check-sigtests-release -Dfail.on.error=true
+        
+      - name: Run gen-sigtests-release
+        if: success() || failure()
+        run: ant $OPTS gen-sigtests-release
```

Comment:
@neilcsmith-net  I tried to merge some of the "paperwork jobs" into one job, but RAT fails.
```
/home/runner/work/netbeans/netbeans/scratch/nbbuild/build.xml:2195: Failed Rat test(s):
Unapproved license in 4 file(s) 
extide/gradle/netbeans-gradle-tooling/.gradle/7.4/gc.properties
extide/gradle/netbeans-gradle-tooling/.gradle/buildOutputCleanup/cache.properties
extide/gradle/netbeans-gradle-tooling/.gradle/vcs-1/gc.properties
extide/gradle/netbeans-gradle-tooling/.gradle/7.4/dependencies-accessors/gc.properties
Unapproved license in 1 file(s) 
profiler/lib.profiler/release/BUILDINFO.txt
Unapproved license in 1 file(s) 
```

I don't know much about RAT. Does it have to be run on a repo which is not built?

Reply:
@mbien yes, RAT should be on clean checkout. Pros and cons to exceptions for build locations. Key thing is source zip is clean.

Reply:
roger. The same is probably also true for sigtests. So I can run the entire job in parallel to the build job.

Reply:
actually, sigtests require a build - but i will figure this out

Reply:
@mbien maybe run RAT as part of the build job? If there's a RAT failure argument for just treating it as a failing build and bailing on everything else.

Reply:
@neilcsmith-net that would be also good i think. Right now I try it with a paperwork job which runs after build and does:
 - extract build artifact
 - print commit delta (allows us to check commit headers of new contributors)
 - check-sigtests-release
 - reset workspace with: git clean -xfdq
 - verify-libs-and-licenses
 - rat
 - gen-sigtests-release

This merges multiple jobs into one, lets see if this works

Reply:
nevermind, this won't work since the build is actually not a git repository at this point anymore. So this will have to run as part of the build as you said.